text/0008-scoped-gems.md

```diff
@@ -118,26 +136,58 @@ and `ensure_gem_scope` might look something like:
+
+Legacy applications that install `hola` will also download and install `hola@mullermp`. In code, `hola` can still be required and used as normal. Attempting to require `hola@mullermp` will return false as it is already loaded.
+
+```
+irb(main):001:0> require 'hola'
+=> true
+irb(main):002:0> Hola.hi
+Hello world!
+=> nil
+irb(main):003:0> require 'hola@mullermp'
+=> false
+```
+
 # Drawbacks
 
-One main drawback is confusion between normal gems and scoped gems. It is important that we update the [recommendations](https://guides.rubygems.org/name-your-gem/) to explain the scoped gems feature and what Ruby namespaces to expect.
+One main drawback is confusion between normal gems and scoped gems. It is important that we update the [recommendations](https://guides.rubygems.org/name-your-gem/) to explain the scoped gems feature and what Ruby namespaces to expect. RubyGems.org does not enforce Ruby namespace with gem naming conventions, nor should it. The `hola@mullermp` gem is expected to have a `Hola` namespace, just as `net-http-digest_auth@ruby` (a potential Ruby official gem) would have `Net::HTTP::DigestAuth`.
 
-An obvious drawback is, with widened gem name availability, there may be more namespace conflicts. Consider a case where user1 creates `configuration@user1` and user2 creates `configuration@user2`, both creating a `Configuration` class. If user 3 installs and requires both, then there may be side effects. This behavior exists today albeit gem names may be different.
+An obvious drawback is, with widened gem name availability, there may be more namespace conflicts. Consider a case where org1 creates `configuration@org1` and org2 creates `configuration@org2`, both creating a `Configuration` class. If a user installs and requires both, then there may be side effects. This behavior exists today albeit gem names may be different. Developers will need to continue using discretion.
 
 # Rationale and Alternatives
 
 The main benefits of this approach are:
 
 1. It is a frugal change to the RubyGems gem specification that is mostly a naming standard. It removes validation around gem naming on `gem build` and `gem install`, and adds validation for first-time gem publishing via `gem push`.
 2. Avoids changes to the Ruby library (and its versions) and how gems are required.
-3. Similar to reserved prefixes for verified publishers (like [NuGet](https://github.com/NuGet/Home/wiki/NuGet-Package-Identity-Verification)) but it can be considered as a *reserved suffix*.
-4. Possible removal of “blacklisted” gem names as important gem names can be migrated to scopes, i.e. `socket@ruby`.
+3. Similar to reserved prefixes for verified publishers (like [NuGet](https://github.com/NuGet/Home/wiki/NuGet-Package-Identity-Verification)). This approach can be considered as a *reserved suffix*.
+4. Eventual removal of "blacklisted" gem list as important standard Ruby gem names are migrated to scopes (i.e. `socket@ruby`, no need to reserve `socket`).
```

Comment:
It's a nice idea to publish standard library gems under a scope like `ruby`.

Reply:
Freeing the top level scope is a nice idea conceptually, but realistically it’s never going to happen because of the risks. Letting a rando claim the unscoped name of a standard library isn’t viable for UX and security reasons, even if it is made viable technically.

Reply:
Ah yep fair point. I wasn't really advocating for actually freeing up existing scope, but instead ensuring that we have an "official" scope for Ruby standard library packages.

Reply:
I think this benefit is certainly optional. My thinking was, existing Ruby standard gems can be migrated to `name@ruby` (using shim), and assuming Ruby standard gems are always pushed to `@ruby` scope, there is no need to block un-scoped equivalent. Ruby team could also register the un-scoped gem and forward it to the scoped version, too.